torch/testing/_internal/common_fsdp.py

```diff
@@ -481,25 +799,26 @@ def _train_for_several_steps(
                     parameter data type is float32."
             else:
                 # FSDP loss is fp16, DDP AMP loss is fp32
-                if isinstance(model, FullyShardedDataParallel):
+                if isinstance(model, FSDP):
                     self.assertEqual(loss.dtype, mixed_precision.param_dtype)
                 else:
                     self.assertEqual(loss.dtype, torch.float32)
             model.module.run_backward(loss)
             if norm_type is not None:
-                if isinstance(model, FullyShardedDataParallel):
-                    model.clip_grad_norm_(clip_norm, norm_type)
+                max_norm = 0.3
```

Comment:
I removed `max_norm` from the method arguments because no caller ever passes it, so it always takes its default value of 0.3. I also renamed to `max_norm` and `norm_type` to match the API.